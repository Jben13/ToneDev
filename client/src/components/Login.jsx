import React from 'react';

class Login extends React.Component {
  constructor() {
    super();
    this.state = {};
  }

  render() {
    return (
      <div>
        <h1>ToneDev</h1>
        <br />
        <button type="submit">
          <a className="login-button" href="/auth/github">Login with GitHub</a>
        </button>
      </div>
    );
  }
}

export default Login;
