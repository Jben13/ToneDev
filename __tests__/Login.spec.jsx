import React from 'react';
import Login from '../client/src/components/Login';

test('App renders component correctly', () => {
  const component = global.shallow(<Login />);
  expect(component).toMatchSnapshot();
});
