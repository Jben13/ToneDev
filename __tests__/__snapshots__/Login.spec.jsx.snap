// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders component correctly 1`] = `
<div>
  Login with GitHub
  <br />
  <a
    href="/auth/github"
  >
    Login with GitHub
  </a>
</div>
`;
